### DISCUSSION TITLE
PySimpleGUI UserSettings & no issues are available here on this GitHub?

### DISCUSSION THREAD

**Discussion by PySimpleGUI** [MeblIkea/Water-Launcher#4] [Ideas]
Hi!

Thank you for posting a screenshot in your readme!  I  ![image](https://user-images.githubusercontent.com/46163555/184515687-56376660-fa55-4d31-8ca0-85cf1d853621.png) seeing what's been created.  It's highly motivating.

I noticed you use JSON to store "settings".  Did you know about the PySimpleGUI UserSettings feature?

I see that this repo doesn't have Issues enabled.  I've never seen a repo that has this turned off.

**Comment by MeblIkea** [comment]
> Hi!
> 
> Thank you for posting a screenshot in your readme!  I  ![image](https://user-images.githubusercontent.com/46163555/184515687-56376660-fa55-4d31-8ca0-85cf1d853621.png) seeing what's been created.  It's highly motivating.
> 
> I noticed you use JSON to store "settings".  Did you know about the PySimpleGUI UserSettings feature?
> 
> I see that this repo doesn't have Issues enabled.  I've never seen a repo that has this turned off.  
> 

Hi, thanks for watching my stuff, I really appreciate it.
PySimpleGUI is a really good Ui module for Python (it's the first and only Ui module I use for Python, because it's really simple to do stuff, and have a cool result.)

I'll try to do your advice, thanks again!

**Reply by MeblIkea** [reply]
Also, I have a suggestion, it can be cool in the function for getting a preview of every theme, that it return which theme we clicked on. It can be easier for user to select their theme.

**Comment by PySimpleGUI** [comment]
There is a function.  It just doesn't return the one you clicked on.  But it does show you the **name** right on the theme itself.   It's described in the docs.

**Comment by PySimpleGUI** [comment]
> Hi, thanks for watching my stuff, I really appreciate it.

Wasn't watching... was just cruising through GitHub, stopped in, and took a quick look at your code.
